src/lib/main/snippet.ts

```diff
@@ -96,7 +96,7 @@ export function snippet(
     doc.querySelector(config!.sandboxParent || 'body')!.appendChild(sandbox);
   }
 
-  function fallback(i?: number, script?: HTMLScriptElement) {
+  function fallback() {
```

Comment:
parameters were never passed to the function

Reply:
Yup you are right. Thanks @aaa123eee